### PR TITLE
Lots of new features

### PR DESCRIPTION
**Pull request by Dimenpsyonal**
_taps clipboard_ (again)

## NEW STUFF!
Settings: Added support for custom Kick messages (Ban, Trelloban, TimeBan, GameBan, Slock, and Whitelist)
Loader: Rescripted it so it looks a bit cleaner
Message: Added the ability to set a custom SystemMessage Image for the :sm command, which shows up below the title
Modified the server to support the new Loader
Modified the UpdateData function to (not break) work better
Updated the Functions module to support the new Kick messages
Updated the Fly script to support using R and F to increase / decrease flight speed.
Aaand that's about it.

### CODE REVIEW

**Comment by Dimenpsyonal** [comment]
probably will never get merged but oh well! i like to try

File: Loader/Loader/Loader.server.lua [hunk 1]
```diff
@@ -1,22 +1,26 @@
...
 
+	NIGHTLY MODULE:
+	https://www.roblox.com/library/8612978896/Nightlies-Adonis-MainModule
 
+--]]
 
-----------------------------------------------------------------------------------------
---                                  Adonis Loader                                     --
-----------------------------------------------------------------------------------------
---		   	  Epix Incorporated. Not Everything is so Black and White.		   		  --
```
[the remark this answers is not in the document]
Comment (Dimenpsyonal) [reply Loader/Loader/Loader.server.lua]: Not everything is so black and white.

**Comment by cocooo23** [comment]
u should add back the Not Everything is so Black and White thats the best part of adonis

**Comment by cocooo23** [comment]
oh its still there good

**Comment by Dimenpsyonal** [comment]
Yes

File: Loader/Config/Settings.lua [hunk 2]
```diff
@@ -284,9 +284,26 @@ settings.CodeExecution = false			-- Enables the use of code execution in Adonis;
...
+	{time} - Shows the Time they were banned, in UTC time
+	{name} - Shows the Player's name
+	{id} - Shows the Player's UserId
+	{moderator} - Shows the moderator who banned the Player
+	{type} - Shows the type of ban, Can display: "TIMEBAN", "SERVERBAN", "GAMEBAN", "CONFIGBAN", "TRELLOBAN"
+	{expiretime} - Shows the time a ban will expire, only applicable on a Timeban, else will return "Undefined" or "Permanent"
+	{remainingtime} - Shows the remaining time on a Timeban, any other form of ban will return "Undefined" or "Permanent"
+--]]
+	
 settings.BanMessage = "Banned"				-- Message shown to banned users upon kick
```
Comment (ccuser44) [inline Loader/Config/Settings.lua:299]: Change this to `You have been banned from this server for {reason}.`

**Review by ccuser44 — CHANGES_REQUESTED** [review]

File: Loader/Config/Settings.lua (hunk 2 above)
Comment (Dimenpsyonal) [reply Loader/Config/Settings.lua:299]: Idk about changing an iconic feature of Adonis

Comment (ccuser44) [reply Loader/Config/Settings.lua:299]: Is it really that iconic?

Comment (Dimenpsyonal) [reply Loader/Config/Settings.lua:299]: obv

**Comment by Dimenpsyonal** [comment]
Good news gang this is almost done

**Comment by Dimenpsyonal** [comment]
ah great a successful check 🥳

**Comment by Dimenpsyonal** [comment]
wow! cant believe we used to have a command file instead of a folder...

[where I got the pull stuff from btw](https://github.com/Epix-Incorporated/Adonis/pull/274)

**Comment by Dimenpsyonal** [comment]
epic linter win

**Comment by Dimenpsyonal** [comment]
apparently i cant add this because big PRs are "bad". How mean

**Pull request closed without merging**